packages/custom-functions-metadata/test/cases/requiresstreamaddress/functions.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+/**
+ * Test the requiresStreamAddress tag
+ * @param x {string} string
+ * @param handler {CustomFunctions.StreamingInvocation<string>} my handler
```

Comment:
This is not a general handler but an object. We may rename it as `invocation`.

And same issue for other `functions.js` and `functions.ts` files.